source/develop/release-management/features/network/networking-api-security-groups.html.md

```diff
@@ -0,0 +1,512 @@
+---
+title: Networking API security groups support
```

Comment:
I'd prefer to call this

 `ovirt-provider-ovn security groups`

because "Networking API" is a very clear term in the OpenStack world, but it is very vague in oVirt.